src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable quotes */
```

Comment:
### Optional
- [ ] Kindly avoid disable eslint rules, they help us write standard code, for example; use single quote instead of double quotes to fix the eslint rules